tests/integration/qualification/main_test.go

```diff
@@ -26,5 +26,12 @@ var (
 )
 
 func TestMain(m *testing.M) {
-	framework.Main("qualification", m, istio.SetupOnKube(&ist, nil))
+	framework.Main("qualification", m, istio.SetupOnKube(&ist, setupConfig))
+}
+
+func setupConfig(cfg *istio.Config) {
+	if cfg == nil {
+		return
+	}
+	cfg.Values["pilot.env.PILOT_ENABLE_LOCALITY_LOAD_BALANCING"] = "true"
```

Comment:
I deleted this and the tests seem to behave the same (I think.. I was getting failures during clean up for both which could have masked it) which makes me think you are testing the CDS locality lb not EDS?